tests/test_grainstats_minicircle.py

```diff
@@ -12,28 +12,34 @@
 RESOURCES = BASE_DIR / "tests" / "resources"
 
 
-def test_grainstats(minicircle_grainstats: GrainStats, minicircle_grainstats_20220526: pd.DataFrame) -> None:
-    """Test the overall GrainStats class."""
+def test_grainstats_regression(regtest, minicircle_grainstats: GrainStats) -> None:
```

Comment:
Does `regtest` have an input type? Similar to `minicircle_grainstats: GrainStats`?